rmgpy/statmech/torsionTest.py

```diff
@@ -117,11 +117,11 @@ def test_getPartitionFunction_free(self):
         Test the FreeRotor.getPartitionFunction() method
         """
         Tlist = numpy.array([300, 500, 1000, 1500, 2000])
-        Qexplist = numpy.sqrt(8*numpy.pi**3*constants.kB*Tlist *
-                              self.freemode.inertia.value_si)/(self.symmetry*constants.h)
-        for T, Qexp in zip(Tlist, Qexplist):
-            Qact = self.freemode.getPartitionFunction(T)
-            self.assertAlmostEqual(Qexp, Qact, delta=1e-4*Qexp)
+        q_exp_list = numpy.sqrt(8*numpy.pi**3*constants.kB*Tlist *
```

Comment:
spaces around operators, same for the lines below

Reply:
Modified, but keep `numpy.pi**3`, I think it is better to distinguish 'power'.

Reply:
I added spaces around the power operator in my PRs, let decide on a consistent approach (whatever everyone agrees on)